modules/civicrm_rules/civicrm_rules_utils.inc

```diff
@@ -42,7 +42,13 @@
  */
 function _civicrm_get_groups() {
   civicrm_initialize();
-  $groupInformation = civicrm_api('group', 'get', array('version' => 3));
+  $groupInformation = civicrm_api('group', 'get', array(
+    'version' => 3,
```

Comment:
Tested this and it works fine. Maybe, its good to write `civicrm_api3()` instead of passing `version` as a param?